src/pages/SignUp/SignUp.jsx

```diff
@@ -0,0 +1,156 @@
+import axios from "axios";
+import { useState, useEffect, useRef } from "react";
+import "./SignUp.css";
+
+const SignUp = () => {
+  const inputRef = useRef(null);
+  const SIGN_UP = process.env.REACT_APP_SKORHEAD_SIGNUP;
+  const initialForm = {
```

Comment:
Suggestion: You can keep initialForm outside SignUp function. Because every time the SignUp components gets mounted. intialForm would be created again.